### DISCUSSION TITLE
Custom padding for Mobile versions?

### DISCUSSION THREAD

**Discussion by addych** [mjmlio/mjml#2907] [Q&A] [answered]
Hello, I'm new to email marketing in general. My job requires me to turn a Figma design into an email, and currently I'm using MJML for that. The emails usually have a left/right padding of 70px, but for mobile, that is a lot, and can only fit about 2 words per line. Is there a way for me to have a different padding only on mobile? Or maybe have proportional padding? Thanks!

**Comment by iRyusa** [comment, marked as answer]
Hi, you can use media queries for that. If you're new to coding in general, please start to learn HTML+CSS first as the doc presume you're familiar with that
